hawc/services/nih/pubmed.py

```diff
@@ -310,6 +310,6 @@ def _get_year(cls, tree: ET.Element, dtype) -> Optional[int]:
     def _get_doi(cls, tree: ET.Element, search_string) -> Optional[str]:
         doi = tree.find(search_string)
         if doi is not None:
```

Comment:
maybe better to say `if doi` - that would prevent both None and empty strings

Reply:
I agree, but for some reason that change totally breaks it, and no dois end up being created in the unit tests.